weatherApp/Globals/Constants.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Constants.swift
+//  weatherApp
+//
+//  Created by Dario Nikolic on 06/08/2020.
+//  Copyright © 2020 Dario Nikolic. All rights reserved.
+//
+
+import UIKit
+
+struct Colors {
+    
+    static let grey             = UIColor(red: 133.0/255.0, green: 147.0/255.0, blue: 152.0/255.0, alpha: 1.0)
```

Comment:
All these properties are not formatted properly. It should be:
`static let grey = UIColor(red: 133.0 / 255.0, green: 147.0 / 255.0, blue: 152.0 / 255.0, alpha: 1.0)`
Also, try to use init with hex. There are online tools that convert this to hex and then you just init with:
`UIColor(rgb: 0x859398)`